cmd/logcli/main.go

```diff
@@ -250,7 +250,10 @@ func newQueryClient(app *kingpin.Application) client.Client {
 	app.Flag("query-tags", "adds X-Query-Tags http header to API requests. This header value will be part of `metrics.go` statistics. Useful for tracking the query. Can also be set using LOKI_QUERY_TAGS env var.").Default("").Envar("LOKI_QUERY_TAGS").StringVar(&client.QueryTags)
 	app.Flag("bearer-token", "adds the Authorization header to API requests for authentication purposes. Can also be set using LOKI_BEARER_TOKEN env var.").Default("").Envar("LOKI_BEARER_TOKEN").StringVar(&client.BearerToken)
 	app.Flag("bearer-token-file", "adds the Authorization header to API requests for authentication purposes. Can also be set using LOKI_BEARER_TOKEN_FILE env var.").Default("").Envar("LOKI_BEARER_TOKEN_FILE").StringVar(&client.BearerTokenFile)
-	app.Flag("retries", "How many times to retry each query when getting an error response from Loki. Can also be set using LOKI_CLIENT_RETRIES env var.").Default("0").Envar("LOKI_CLIENT_RETRIES").IntVar(&client.Retries)
+	app.Flag("max-retries", "How many times to retry each query when getting an error response from Loki. Can also be set using LOKI_CLIENT_MAX_RETRIES env var.").Default("1").Envar("LOKI_CLIENT_MAX_RETRIES").IntVar(&client.MaxRetries)
+	app.Flag("min-backoff", "Minimum backoff time between retries. Can also be set using LOKI_CLIENT_MIN_BACKOFF env var.").Default("1").Envar("LOKI_CLIENT_MIN_BACKOFF").IntVar(&client.MinBackoff)
+	app.Flag("max-backoff", "Maximum backoff time between retries. Can also be set using LOKI_CLIENT_MAX_BACKOFF env var.").Default("1").Envar("LOKI_CLIENT_MAX_BACKOFF").IntVar(&client.MaxBackoff)
```

Comment:
In Loki we namespace the parameters. We could to the same here.`backoff.max-retries`, `backoff.min-period` and `backoff.max-period`.

Reply:
fixed